src/game.js

```diff
@@ -0,0 +1,91 @@
+import Player from './player';
+import * as Board from './board';
+
+function gameOver(status, playerX, playerO) {
+  const gameStatus = document.getElementById('game-status');
+  const slots = document.getElementsByClassName('slot');
+
+  for (let i = 0; i < slots.length; i += 1) {
+    slots[i].classList.remove('clickable');
+  }
+
+  if (playerX.winner) {
+    gameStatus.innerText = `${playerX.name} wins!`;
+  } else if (playerO.winner) {
+    gameStatus.innerText = `${playerO.name} wins!`;
+  } else {
+    gameStatus.innerText = 'It is a draw!';
+  }
+
+  return null;
+}
+
+function checkWinStatus(status, player) {
+  if (status === player.symbol) player.winner = true;
+
+  return null;
+}
+
+function start(playerXName, playerOName) {
+  // I am going to delete all the slots to clear the board
+  const gameBoard = document.getElementById('board-container');
+
+  while (gameBoard.children.length !== 0) {
+    gameBoard.removeChild(gameBoard.firstChild);
+  }
+
+  // Clear the board array
+  Board.clear();
+  // After getting the players, we render the board
+  Board.render();
+  const playerX = Player(playerXName, 'x');
+  const playerO = Player(playerOName, 'o');
```

Comment:
[optional] When the  game starts without a player typing a name it is difficult to know who is playing, there should be a default, maybe player1 and player2, this default would be used when a name is not entered

Reply:
Seems reasonable.